docs/_docs/04_reference/02-cli-options.md

```diff
@@ -196,202 +186,226 @@ The algorithm for determining the aggregate exit code is as follows:
 - If any unit throws a 2, but nothing throws a 1, Terragrunt will throw a 2.
 - If nothing throws a non-zero, Terragrunt will throw a 0.
 
-### plan-all (DEPRECATED: use run-all)
+#### graph
 
-**DEPRECATED: Use `run-all plan` instead.**
+Run the provided OpenTofu/Terraform command against the graph of dependencies for the unit in the current working directory. The graph consists of all units that depend on the unit in the current working directory via a `dependency` or `dependencies` blocks, plus all the units that depend on those units, and all the units that depend on those units, and so on, recursively up the tree, up to the Git repository root, or the path specified via the optional `--graph-root` argument.
 
-Display the plans of a `stack` by running `terragrunt plan` in each subfolder. Make sure to read [Execute OpenTofu/Terraform
-commands on multiple modules at once](/docs/features/stacks) for
-context.
+The Command will be executed following the order of dependencies: so it'll run on the unit in the current working directory first, then on units that depend on it directly, then on the units that depend on those units, and so on. Note that if the command is `destroy`, it will execute in the opposite order of the dependencies.
```

Comment:
```suggestion
The Command will be executed following the order of dependencies: so it'll run on the unit in the current working directory first, then on units that depend on it directly, then on the units that depend on those units, and so on. Note that if the command is `destroy`, it will run in the opposite order of the dependencies (the final dependents, then their dependencies, etc. up to the unit you ran the command in).
```